sdk/textanalytics/ai-text-analytics/src/generated/models/index.ts

```diff
@@ -924,11 +924,17 @@ export interface AnalyzeTextCancelJobHeaders {
 
 /** Known values of {@link AnalyzeTextTaskKind} that the service accepts. */
 export enum KnownAnalyzeTextTaskKind {
+  /** SentimentAnalysis */
```

Comment:
Does this help with intellisense or something? It doesn't seem like these comments are all that necessary, but correct me if I'm wrong

Reply:
They only needed to make the tool we use to generate the public API happy. If you look at the document under the review folder, you'll see a lot of `undocumented` comments removed. Reducing the number of these comments should help catch undocumented definitions.